scripts/prepare/bundle.ts

```diff
@@ -80,6 +80,7 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
           : false,
       esbuildOptions: (c) => {
         /* eslint-disable no-param-reassign */
+        c.conditions = ['module'];
```

Comment:
I would recheck what kind of an effect does it have on esbuild - is this purely additive to the default conditions? or does it replace them? I think that I was checking this in the past and that this is additive - some default conditions just cant be turned off but I'm not 100% sure.